DigitalLearningSolutions.Web/Views/LearningMenu/Section/_TutorialCard.cshtml

```diff
@@ -3,22 +3,25 @@
 <div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
   <div class="nhsuk-card__content">
     <div class="nhsuk-grid-row">
-      <div class="@(Model.ShowLearnStatus ? "nhsuk-grid-column-three-quarters" : "nhsuk-grid-column-full")">
-        <h3 class="nhsuk-card__heading" id="@Model.Id-name">
+      <div class="nhsuk-grid-column-three-quarters">
+        <h3 class="nhsuk-card__heading completion-card-heading" id="@Model.Id-name">
```

Comment:
This was something that benji wrote for the completion card and i needed to use the same styling. He was at the time working on extracting the partial so if I made some changes they would cause a lot of conflicts. I should be able to merge learning-menu-master into this now and make those changes as he is now done with that